src/util/kafka/StreamHandler.h

```diff
@@ -23,7 +23,7 @@ limitations under the License.
 
 class StreamHandler {
  public:
-    StreamHandler(KafkaConnector *kstream, int numberOfPartitions, std::vector<DataPublisher *> &workerClients);
+    StreamHandler(KafkaConnector *kstream, int numberOfPartitions, std::vector<DataPublisher *> &workerClients, SQLiteDBInterface* sqlite);
```

Comment:
Can you break this line into two? We need to ensure the line length of less than 120 characters.

Reply:
I broke this line

Reply:
Ok